jenkins/slave-base/Dockerfile

```diff
@@ -9,9 +9,10 @@ ENV SONAR_SCANNER_VERSION=3.1.0.1141 \
     SKOPEO_VERSION=0.1.37-3 \
     OSTREE_VERSION=2018.5-1
 
-ARG APP_DNS=192.168.99.100.nip.io
-ARG APP_DNS_PORT=443
-ARG SNYK_DISTRIBUTION_URL
+ARG APP_DNS=
+ARG APP_DNS_PORT=
```

Comment:
@oalyman I left these for now. Did you already check if we can remove them using the approach you've chosen for the Jenkins master, SonarQube, etc.?